bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -221,6 +222,11 @@ impl SlidingSyncRoom {
                     Err(RecvError::Closed) => break,
                     Ok(_) | Err(RecvError::Lagged(_)) => timeline.clear().await,
                 }
+
+                match ignore_user_list_rx.recv().await {
```

Comment:
Another idea, now that we know we need to reset SS as well, would be to merge both into the `sliding_sync_reset_broadcast`

Reply:
at most what I could so is just have the receive of the ignore, trigger a send on the `sliding_sync_reset_broadcast`